crypto/fipsmodule/ec/p256-nistz.c

```diff
@@ -185,7 +186,50 @@ static void ecp_nistz256_mod_inverse_sqr_mont(BN_ULONG r[P256_LIMBS],
   ecp_nistz256_sqr_mont(r, ret);  // 2^256 - 2^224 + 2^192 + 2^96 - 2^2
 }
 
+#endif
+
+
+// r = p * p_scalar
+
+#if defined(EC_P256_USE_S2N_BIGNUM)
+
+static void ecp_nistz256_windowed_mul(const EC_GROUP *group, P256_POINT *r,
+                                      const EC_JACOBIAN *p,
+                                      const EC_SCALAR *p_scalar) {
+  uint64_t s2n_point[12], s2n_result[12];
+
+  assert(p != NULL);
+  assert(p_scalar != NULL);
+  assert(group->field.N.width == P256_LIMBS);
+
+  OPENSSL_memcpy(s2n_point,p->X.words,32);
+  OPENSSL_memcpy(s2n_point+4,p->Y.words,32);
+  OPENSSL_memcpy(s2n_point+8,p->Z.words,32);
```

Comment:
Can some of this use constants/known sizes? I think this 32 could be `sizeof(p->X.words)`, also would be handy to figure out what the 4 and 8 are coming from. Also spacing is a little off, convention is spaces after punctuation/symbols:
```suggestion
  OPENSSL_memcpy(s2n_point, p->X.words, sizeof(p->X.words));
  OPENSSL_memcpy(s2n_point + ????,p->Y.words, sizeof(p->Y.words));
  OPENSSL_memcpy(s2n_point + ????,p->Z.words, sizeof(p->Z.words));
```